docs/UsersGuide/source/Introduction.rst

```diff
@@ -210,44 +363,21 @@ A list of available documentation is shown in :numref:`Table %s <list_of_documen
    +----------------------------+---------------------------------------------------------------------------------+
 
 The UFS community is encouraged to contribute to the development effort of all related
-utilities, model code, and infrastructure. Issues can be posted in the GitHub repository
-for the SRW Application or the relevant subcomponent to report bugs or to announce upcoming
-contributions to the code base. For code to be accepted in the authoritative repositories,
-the code management rules of each component (described in the User’s Guides listed in
-:numref:`Table %s <list_of_documentation>` need to be followed.
+utilities, model code, and infrastructure. Users can post issues in the related GitHub repositories to report bugs or to announce upcoming contributions to the code base. For code to be accepted in the authoritative repositories, users must follow the code management rules of each component, which are outlined in the respective User's Guides listed in :numref:`Table %s <list_of_documentation>`.
 
 Future Direction
-================
+=================
 
-Users can expect to see incremental improvements and additional capabilities in upcoming
-releases of the SRW Application to enhance research opportunities and support operational
-forecast implementations. Planned advancements include:
+Users can expect to see incremental improvements and additional capabilities in upcoming releases of the SRW Application to enhance research opportunities and support operational forecast implementations. Planned enhancements include:
 
 * A more extensive set of supported developmental physics suites.
 * A larger number of pre-defined domains/resolutions and a fully supported capability to create a user-defined domain.
 * Inclusion of data assimilation, cycling, and ensemble capabilities.
-* A verification package (i.e., METplus) integrated into the workflow. 
+* A verification package (e.g., METplus) integrated into the workflow. 
 * Inclusion of stochastic perturbation techniques.
 
-In addition to the above list, other improvements will be addressed in future releases.
-
-
-How to Use This Document
-========================
-
-This guide instructs both novice and experienced users on downloading,
-building and running the SRW Application.  Please post questions in the
-UFS forum at https://forums.ufscommunity.org/.
-
-.. code-block:: console
-
-   Throughout the guide, this presentation style indicates shell
-   commands and options, code examples, etc.
+.. bibliography:: references.bib
```

Comment:
We will be supporting the capability to create a user-defined domain in the v2 release, although I'm not sure I would call it a "fully supported capability" quite yet.  There are still a lot of manual changes a user needs to make.  DA and cycling are coming, so that's good.  The ensemble capability already exists.  Vertification is now available and will be supported for both deterministic and ensemble simulations, and while we previously included tendency-based stochastic physics (SPPT, SHUM, and SKEB) in the v1 release, Stochastically Perturbed Perturbations (SPP) will also now be included, and all four schemes will now be supported, where none were supported in v1.

Reply:
You could also add that user-defined vertical levels (number and distribution) will be supported in an upcoming release.

Reply:
Made these changes. Are there other specific improvements planned for the future? I left the grid bullet point in, since I don't think we have expanded the number of predefined grids, and, as you say, the user-defined capability isn't _fully_ supported, even though it was in option in the last release and has expanded support in this one. I can remove it though if you prefer.

Reply:
The reason I wasn't sure about "fully supported" for the user-defined domain capability is that it's the same as it was in version 1.0.0.  In other words, the user still needs to make a significant amount of manual modifications to create their own domain in the set_predef_grid_params.sh script.  If EPIC is confident they can help users with this feature, then it could be considered fully supported.  At some point, it would be preferable to introduce config.sh-based options that a user could leverage to introduce a new pre-defined domain.